src/applications/vaos/containers/ContactInfoPage.jsx

```diff
@@ -1,9 +1,16 @@
 import React from 'react';
 import { connect } from 'react-redux';
-import { openFormPage, updateFormData } from '../actions/newAppointment.js';
 import SchemaForm from 'platform/forms-system/src/js/components/SchemaForm';
-import ProgressButton from 'platform/forms-system/src/js/components/ProgressButton';
 import phoneUI from 'platform/forms-system/src/js/definitions/phone';
+import FormButtons from '../components/FormButtons';
```

Comment:
The form page changes I made were to use the common components and also a common selector I created for form page data.